l10n/en-US/viewer.properties

```diff
@@ -41,6 +41,10 @@ print.title=Print
 print_label=Print
 save.title=Save
 save_label=Save
+# LOCALIZATION NOTE (download.title): This string is used in Firefox for Android.
+download.title=Download
```

Comment:
It would be great to use semantic IDs. Download as a title is different from download as a button label, as in noun vs verb, and that's information needed by localizer to translate.

Can we use `download_button` instead of `download`? In any case, please add the information in the comment (for both strings).